tests/pausing_test.go

```diff
@@ -51,10 +51,12 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 	var err error
 	var virtClient kubecli.KubevirtClient
+	var namespace string
 
 	BeforeEach(func() {
 		virtClient, err = kubecli.GetKubevirtClient()
 		util.PanicOnError(err)
+		namespace = util.NamespaceTestDefault
```

Comment:
Does this variable ever vary? If it's just for shorthand, can we use a local constant?

Reply:
So I believe it has to be this way as each worker/node gets different namespaces and making it const would be compile-time, right?

Reply:
Oh, I was not aware of the magic described in tests/README.md

```
## Test Namespaces

If tests are executed in parallel, every test gets its unique set of namespaces
to test in. If you write a test and reference the namespaces
`test.NamespaceTestDefault`, `test.NamespaceTestAlternative` or
`tests.NamespaceTestOperator`, you get different values, based on the ginkgo
execution node.

For as long as test resources are created by referencing these namespaces,
there is no test conflict to expect.
```
and I find [it](https://github.com/kubevirt/kubevirt/blob/53cf99ec3/tests/testsuite/namespace.go#L270) way too clever.
Modifying global variables is a well-known anti-pattern. It is probably outside the realm of your PR, but can we have something like `testsuite.DefaultNamespace()` so it is clear to the user that the testsuite may change it?

Reply:
@dankenigsberg Do you mean something like this?
```
package main

import "fmt"

var namespace string = "default"

func ns() string {
	return namespace
}

var localnamespace = ns()

var localnamespace2 = namespace

func main() {
	namespace = "default1"
	fmt.Println("Hello " + namespace)

	fmt.Println("Hi " + localnamespace)

	fmt.Println("Hello there  " + localnamespace2)
}
```

Reply:
I don't think I follow your example with `main()` and a private function `ns()`. What I find troubling is having a function in one package (`testsuite.CalculateNamespaces`) modifying global variables in two packages (`util.NamespaceTestDefault`, `testsuite.util.NamespaceTestDefault`) in timing unbeknown to a naive reader of the code using them.

I think that it would be less surprising to have

```
package testsuite

var worker

func CalculateNamespaces() {
        worker = GinkgoParallelProcess()
}

func DefaultNS() {
        return fmt.Sprintf("%s%d", something, worker)
}

func AlternativeNS() {
        return fmt.Sprintf("%s%d", somethingelse, worker)
}

```

It's nicer when each package is responsible to its own data, and the user of `DefaultNS()` may assume that different calls may return different values.

(If I was not clear, I do not suggest adding something like this to this PR.)